dashboard/app/models/foorm/library.rb

```diff
@@ -14,4 +14,50 @@
 #  index_foorm_libraries_on_multiple_fields  (name,version) UNIQUE
 #
 class Foorm::Library < ApplicationRecord
+  include Seeded
+
+  #to do: make these name and version instead of library_name and library_version
+  has_many :library_questions, primary_key: [:name, :version], foreign_key: [:library_name, :library_version]
+
+  def self.setup
+    Dir.glob('config/foorm/library/**/*.json').each do |path|
+      # Given: "config/foorm/library/surveys/pd/pre_workshop_survey.0.json"
+      # we get full_name: "surveys/pd/pre_workshop_survey"
+      #      and version: 0
+      unique_path = path.partition("config/foorm/library/")[2]
+      filename_and_version = File.basename(unique_path, ".json")
+      filename, version = filename_and_version.split(".")
+      version = version.to_i
+      full_name = File.dirname(unique_path) + "/" + filename
+
+      # Let's load the JSON text.
+      begin
+        library = Foorm::Library.find_or_initialize_by(
```

Comment:
This little chunk (creating the library) is the only difference between what's in LibraryQuestion currently and what I've moved in here.